pkg/telemetry/statsconn.go

```diff
@@ -69,17 +69,17 @@ func NewPacketConn(c net.PacketConn, direction prometheus.Direction) *PacketConn
 func (c *PacketConn) ReadFrom(p []byte) (n int, addr net.Addr, err error) {
 	n, addr, err = c.PacketConn.ReadFrom(p)
 	if n > 0 {
-		prometheus.IncrementBytes(prometheus.Incoming, uint64(n), false)
-		prometheus.IncrementPackets(prometheus.Incoming, 1, false)
+		prometheus.IncrementBytes(prometheus.Incoming, uint64(n), false, "")
+		prometheus.IncrementPackets(prometheus.Incoming, 1, false, "")
 	}
 	return
 }
 
 func (c *PacketConn) WriteTo(p []byte, addr net.Addr) (n int, err error) {
 	n, err = c.PacketConn.WriteTo(p, addr)
 	if n > 0 {
-		prometheus.IncrementBytes(prometheus.Outgoing, uint64(n), false)
-		prometheus.IncrementPackets(prometheus.Outgoing, 1, false)
+		prometheus.IncrementBytes(prometheus.Outgoing, uint64(n), false, "")
+		prometheus.IncrementPackets(prometheus.Outgoing, 1, false, "")
```

Comment:
Can use "edge" instead of empty I think. Would be good to hear more opinions on this.

Reply:
I'm wondering if it should be the same metric? while it's convenient to re-use the same labels. this can get confusing with mixing "region" with "edge". particularly, OSS only uses the latter.

Reply:
Sounds good, I'll create new counters for these.